app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryTask.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import TasksList from '../task/TasksList';
+import PropTypes from 'prop-types';
+
+class ExpandedStoryTask extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      task: ''
+    };
+    this.onInputChange = this.onInputChange.bind(this);
```

Comment:
Do you know why this was necessary? I'm not saying it's not, but asking just to be sure.

Reply:
yes I do, the  `bind` will connect the method `onInputChange` with _instance_